run/make_init_rect10x10.py

```diff
@@ -24,6 +26,10 @@
 hice[:,:] = 0.1
 hsnow = datagrp.createVariable("hsnow", "f8", hfield_dims)
 hsnow[:,:] = 0.0
+<<<<<<< issue275_2_arrayordering
 tice = datagrp.createVariable("tice", "f8", ("nLayers", "y", "x",))
+=======
+tice = datagrp.createVariable("tice", "f8", ("nLayers", "y", "x"))
+>>>>>>> 02453ff Update the basic 10x10 grid to the swapped array ordering.
```

Comment:
I think you have left some merge conflict lines in this one.

Reply:
Fixed